es6/executor.js

```diff
@@ -1,16 +1,19 @@
 /** @const {boolean} exists check whether nOS is defined */
-export const exists = !!window.NOS && !!window.NOS.V1;
+const exists = !!window.NOS && !!window.NOS.V1;
```

Comment:
We actually did export this for users to have a mechanic to check if the nOS API is injected or not. I think many people are depending on this :P

Reply:
Yeah `exists` was exported into the basic nOS namespace and now it's gone. Might be a nifty built-in trick that we should keep.